scripts/backport.py

```diff
@@ -48,10 +48,9 @@ def get_referenced_issue(pr_number):
             repository(owner: "timescale", name: "timescaledb") {
               pullRequest(number: $pr_number) {
                 closingIssuesReferences(first: 1) {
-                  edges {
-                    node {
-                      number
-                    }
+                  nodes {
+                      number, title,
+                      labels (first: 30) { nodes { name } }
```

Comment:
Could you document why the limitation of 30 was chosen?

Reply:
Just some arbitrary number, it is required here by GitHub. It should be not very big because it's counted towards the usage quota of the GraphQL API. I'll add a comment there.